src/lread.c

```diff
@@ -4009,6 +4009,20 @@ check_obarray (Lisp_Object obarray)
   return obarray;
 }
 
+/* Like check_obarray, but for the global Vobarray. */
```

Comment:
it looks like Vobarray is defined as `#define Vobarray globals.f_Vobarray` in globals.h. We set up the emacs globals as remacs_sys::globals, so it seems like if we need access to the `V...` variables in Rust, we can just use the global object (unless I am misunderstanding the situation).